fxapom/pages/sign_in.py

```diff
@@ -57,7 +57,7 @@ def check_for_popup(self, handles):
             self.popup = True
             for handle in handles:
                 self.driver.switch_to.window(handle)
-                if self.is_element_visible(*self._fox_logo_locator):
+                if self.is_element_present(*self._fox_logo_locator):
```

Comment:
This is intended for determining if the window is the FxA popup. The logo isn't immediately visible, so we need to check for presence instead. There may be a better way to do this, but this restores the previous functionality, which was regressed when removing implicit waits.